Marlin/Marlin_main.cpp

```diff
@@ -8434,6 +8435,21 @@ inline void gcode_M211() {
   SERIAL_ECHOLNPAIR(" " MSG_Z, soft_endstop_max[Z_AXIS]);
 }
 
+/**
+ * M217: Send text to serial output
+ *
+ * Usage: M217 some text
+ * Text must be prefixed by "//" and this will be prepended if it is not present
+ */
+inline void gcode_M217() {
+  if (strncmp(parser.string_arg, "//", 2) == 0) {
+    SERIAL_ECHOLN(parser.string_arg);
+  } 
+  else {
+    SERIAL_ECHOLNPAIR("// ", parser.string_arg);
+  } 
+}
```

Comment:
Hey guys, not sure whether i'm in the right place or not but since i uploaded the new marlin firmware i cant receive M114, M117, M119 and other m-code reports through my wireless connection from my printer to pronterface. (I have newest marlin, mks 1.5, mks-tft 2.0.0)

I've uploaded your M217 code however unfortunately this doesn't seem to be the solution either.. any ideas how to get the reports of endstop status and extruder positioning back? or is there a simple work around??